modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -542,15 +549,15 @@ private void handleClusterState(ClusterStateMessage msg, String senderConsistent
 
     private CompletableFuture<CmgRaftService> completeJoinIfTryingToRejoin(CmgRaftService cmgRaftService) {
         if (attemptedCompleteJoinOnStart) {
-            return cmgRaftService.completeJoinCluster()
+            return cmgRaftService.completeJoinCluster(nodeAttributes.nodeAttributes().value())
                     .thenApply(unused -> cmgRaftService);
         } else {
             return completedFuture(cmgRaftService);
         }
     }
 
     private CompletableFuture<CmgRaftService> joinCluster(CmgRaftService service, ClusterTag clusterTag) {
-        return service.startJoinCluster(clusterTag)
+        return service.startJoinCluster(clusterTag, nodeAttributes.nodeAttributes().value())
```

Comment:
Discussed in private, that was made because we want to have only `LogicalNode` in `org.apache.ignite.internal.cluster.management.topology.LogicalTopology`, so we have to pass this attributes for start as well.